daily-challenges/2024-11/10/ththao27.spec.ts

```
@@ -0,0 +1,46 @@
+interface StepsAnalysis {
+    totalSteps: number;        // Tổng số bước trong tuần
+    averageSteps: number;      // Trung bình mỗi ngày
+    bestDay: number;           // Số bước cao nhất
+    worstDay: number;          // Số bước thấp nhất
+    daysAboveTarget: number;   // Số ngày đạt mục tiêu
+    streak: number;            // Số ngày liên tiếp đạt mục tiêu
+}
+
+function analyzeSteps (dailySteps: number[], target: number = 10000): StepsAnalysis {
+    const totalSteps = dailySteps.reduce((sum, currentValue) => sum + currentValue, 0);
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)